jcl/src/java.base/share/classes/java/lang/System.java

```diff
@@ -378,7 +378,9 @@ private static void ensureProperties() {
 	systemProperties.put("sun.jnu.encoding", platformEncoding); //$NON-NLS-1$
 	systemProperties.put("file.encoding", fileEncoding); //$NON-NLS-1$
 	systemProperties.put("file.encoding.pkg", "sun.io"); //$NON-NLS-1$ //$NON-NLS-2$
+	/*[IF !Java12]*/
```

Comment:
@JasonFengJ9 Can you confirm that this means that this is exclude for 12+?  I always get the java preprocessor tags confused :)

Reply:
@DanHeidinga yes, that's correct. 
`!Java12` means it excludes the content decorated within `pConfig` specified with JPP flag `Java12`, i.e., `12+` assuming all future Java version pConfig specify `Java12` as well.

Reply:
Thanks for the confirmation!